drivers/reset/Kconfig

```diff
@@ -0,0 +1,30 @@
+# Reset Controller driver configuration options
+
+# Copyright (c) 2022 Andrei-Edward Popa <andrei.popa105@yahoo.com>
```

Comment:
Copyright email does not match git signed-off-by email. Is this intentional?

Reply:
Note that this is just a question; I'm not requesting a change for this item.

Reply:
Thanks for pointing that out, I forgot to change the email address for the git commits with my personal email address, now it is fixed.

Reply:
@mbolivar-nordic The documentation is failing because of this:

 Warning, treated as error:
/home/runner/work/zephyr/zephyr/doc/_build/src/releases/release-notes-3.0.rst:655:Unknown interpreted text role "kconfig".

Probably because at line 655 needs to be kconfig:option?

Reply:
rebased

Reply:
Cool! 😊